src/cljs/rems/administration/create_workflow.cljs

```diff
@@ -15,7 +16,7 @@
             [rems.util :refer [navigate! post! put! trim-when-string]]))
 
 (defn- item-by-id [items id-key id]
-  (medley.core/find-first #(= (id-key %) id) items))
+  (medley/find-first #(= (id-key %) id) items))
```

Comment:
I think the approach we chose was to use `:refer` for Medley because it's as good as `core`.